contracts/dmarket.js

```diff
@@ -0,0 +1,1258 @@
+/* eslint-disable no-await-in-loop */
+/* global actions, api */
+
+const UTILITY_TOKEN_SYMBOL = 'BEE';
+
+// 30 days in seconds
+const MAX_EXPIRATION_SECS = 2592000;
+// fee in UTILITY TOKEN required to create a new pair
+const PAIR_CREATION_FEE = '500';
+
+const CONTRACT_NAME = 'dmarket';
+
+actions.createSSC = async () => {
+  const tableExists = await api.db.tableExists('quoteTokens');
+
+  if (tableExists === false) {
+    await api.db.createTable('quoteTokens', ['quoteToken']);
+    await api.db.createTable('buyBook', ['symbol', 'quoteToken', 'account', 'priceDec', 'expiration', 'txId']);
```

Comment:
[{ name: 'byPriceDecDesc', index: { 'quoteToken': 1, 'symbol': 1, 'priceDec': -1 }}, 'expiration']  appear to be the only indexes needed.

Reply:
I do not see any queries by txId, but please confirm.

Reply:
[{ name: 'byPriceDecAsc', index: { 'quoteToken': 1, 'symbol': 1, 'priceDec': 1}}, 'expiration']